charts/ks-devops/charts/jenkins/values.yaml

```diff
@@ -1,4 +1,11 @@
 Master:
+  StatefulSet:
+    Enable: true
+    Replicas: 1
+    # StatefulSetUpdateStrategy must be 'RollingUpdate' or 'OnDelete'
+    UpdateStrategy: RollingUpdate
```

Comment:
`OnDelete` may be reasonable, because there is only one Jenkins instance forever.

Reply:
@LinuxSuRen what do you reckon to this strategy.

Reply:
If this stategy is set to "OnDelete", an error will occur when executing "helm upgrade"

Reply:
Okay, I will test against it.